cmds/kcl-go/command/cmd_doc.go

```diff
@@ -11,7 +11,7 @@ import (
 	"kusionstack.io/kclvm-go/pkg/kclvm_runtime"
 )
 
-func NewDocCmd() *cli.Command {
+func newDocCmd() *cli.Command {
```

Comment:
Why change these functions to private, which seems to be related to this PR

Reply:
reverted